Board.java

```diff
@@ -0,0 +1,634 @@
+package test;
+import java.util.ArrayList;
+
+public class Board {
+    public class BoardSquare {
```

Comment:
inner class is not a good practice

Reply:
thanks, will extract it to a different file